account_avatax/models/avalara_salestax.py

```diff
@@ -165,6 +165,7 @@ def get_avatax_rest_service(self):
             self.service_url,
             self.request_timeout,
             self.logging,
+            config=self,
```

Comment:
I assume this is added to avoid error on a call somewhere.
The proper way to go is to fix the call, and not the function signature here.
Thoughts?

Reply:
https://github.com/OCA/account-fiscal-rule/blob/13.0/account_avatax/models/avalara_salestax.py#L309
https://github.com/OCA/account-fiscal-rule/blob/13.0/account_avatax/models/partner.py#L158

We could have the same parameters in this method too i.e. config=self

Reply:
We don't need to specify the avatax info as it is obtained from the config file in https://github.com/OCA/account-fiscal-rule/blob/13.0/account_avatax/models/avatax_rest_api.py#L20

Reply:
OK